src/lib/es5.d.ts

```diff
@@ -1743,9 +1743,8 @@ interface Int8Array {
 }
 interface Int8ArrayConstructor {
     readonly prototype: Int8Array;
-    new(length: number): Int8Array;
-    new(array: ArrayLike<number>): Int8Array;

```

Comment:
please keep the first overload there, and change the second to be ` new(arrayOrBuffer: ArrayLike<number> | ArrayBufferLike): Int8Array; `
this way it is clearer that it can either be called with a `number`, or with `ArrayLike`/`ArrayBufferLike`